chatkit-core/src/main/kotlin/com/pusher/chatkit/files/AttachmentTypes.kt

```diff
@@ -4,7 +4,7 @@ import java.io.File
 
 sealed class GenericAttachment
 
-data class DataAttachment constructor(
+internal data class DataAttachment constructor(
```

Comment:
This is v1's public type. It will be redesigned in v2, post MVP. Definitely the whole file should be removed.

Reply:
👍 have removed.